input/LimitStateDesign.tex

```diff
@@ -436,48 +430,46 @@ \section{Unified Approach}
 \label{fig:various}
 \end{figure}
 
-For a section under bending combined with axial compression, clause 39.1 (b) of the Code
-gives the equation,
+For a section under bending combined with axial compression, clause 39.1 (b) of the Code gives the equation,
 \begin{equation}
 \epsilon_1=0.0035-0.75\epsilon_2
 \end{equation}
+
 which represents various strain diagrams given in the line rotating about
-the point A (\fig 1.6). On a similar analbgy, the line rotating about the
+the point A (\fig 1.6). On a similar analogy, the line rotating about the
 point B (\fig 1.6) gives different strain diagrams applicable for a
 section under bending combined with axial tension, which can be got from
 the following relations for different types of reinforcing steel.
 
-\begin{align}
-For 
-Fe 250,\epsilon_{st}=0.0020-5.45\epsilon_1\\
-Fe 415,\epsilon_{st}=0.0145-2.82\epsilon_1\\
-Fe 500,\epsilon_{st}=0.012-1.86\epsilon_1
-\end{align}
+\begin{center}
+For $\fetwofivezero,\epsilon_{st}=0.0020-5.45\epsilon_1$\\
+For $\fefouronefive,\epsilon_{st}=0.0145-2.82\epsilon_1$\\
+For $\fefivezerozero,\epsilon_{st}=0.012-1.86\epsilon_1$
+\end{center}
 
 The above relationships have been derived by using the principles of
-similar triangles along with the values of $\epsilon_{st}, max, \epsilon_{std}$
-for various types of steel given in \tablem 1.2. Figure 1.6 gives by
+similar triangles along with the values of $\epsilon_{st,max}, \epsilon_{std}$ for
+various types of steel given in \tablem 1.2. Figure 1.6 gives by
 similar triangles,
 
 \begin{equation}
 n=\frac{\epsilon_1}{\epsilon_1+\epsilon_{st}}
 \end{equation}
 
-$n_b (balanced), n_max, n_min$ are calculated by the \eqn 1.26 and are
+$n_b (balanced), n_{max}, n_{min}$ are calculated by the \eqn 1.26 and are
 given in \tablem 1.2 for ready reference. Balanced failure is given by
 $\epsilon_1 = 0.0035, \epsilon_{st} =\epsilon_{sy}$ ($\epsilon_{sy}$
 corresponds to a steel strain when $f_{st} = 0.87 f_y$, to be read from
 stress-strain curves for steel given in \fig 23 of the
-\citetitle{is4562000}, balanced failure modiﬁed for ductility is given for
-$\epsilon_1 = 0.0035, \epsilon_{st} = \epsilon-{std}$ and tension failure
+\citetitle{is4562000}, balanced failure modified for ductility is given for
```

Comment:
I tell you. The first word modified was copy paste from .txt file that we have created using OCR. So, it was printed like this "modied" in pdf. you can see there is no space between "fi". So, I write it manually, Now its Ok.  It show complete word modified in pdf.